.github/workflows/frontend-skipped-checks.yml

```diff
@@ -1,11 +1,27 @@
+# Required checks with path filtering rules will block pull requests from merging if they change only the excluded files.
+# This is a workaround to allow the PR to be merged.
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
 name: Frontend
 
 on:
+  push:
+    branches:
+      - 'master'
+      - 'release-**'
+    paths:
+      - "docs/**"
+      - "**.md"
+      - "**.cljc?"
```

Comment:
Is this meant to be `*.clj?` because we want to include the ClojureScript (`.cljs`) changes as well?

Reply:
Good point re: `.cljs` @ariya . Thanks

However, I don't think `**.clj?` is the right pattern. It will match `**.cl` and `**.clj` files.

We most likely need `**.clj*` in this case.